qa/sc_evm_cert_key_rotation.py

```diff
@@ -82,7 +84,6 @@ def secure_enclave_create_signature(message_to_sign, public_key="", key=""):
 
     def run_test(self):
```

Comment:
I'm not sure what check you refer to. Something like this, but for status == success?
```
        generate_next_blocks(sc_node, "first node", 1)
        receipt = sc_node.rpc_eth_getTransactionReceipt("0x" + response['result']['transactionId'])
        status = int(receipt['result']['status'], 16)
        assert_equal(0, status, "Wrong tx status in receipt")
```

Reply:
I will add this check for the first couple of transactions, one for signing and one for master key

Reply:
In my understanding, the execSubmitKeyRotation in CertificateKeyRotationMsgProcessor adds a SubmitKeyRotation event log inside the receipt of the transaction. What I was trying to say is it would be nice to check the correctness of this event log inside the receipt, in addition to the status. You can find some example in `check_withdrawal_event` inside sc_evm_backward_transfer.py file.

Reply:
This check will be added in a separate PR, to not block this one now